src/explosion.cpp

```diff
@@ -417,8 +419,8 @@ static std::vector<tripoint> shrapnel( const tripoint &src, int power,
             frag.proj = proj;
             frag.proj.speed = cloud.velocity;
             frag.proj.impact = damage_instance::physical( 0, damage, 0, 0 );
-            // dealt_dag->m.total_damage() == 0 means armor block
-            // dealt_dag->m.total_damage() > 0 means took damage
+            // dealt_dam.total_damage() == 0 means armor block
+            // dealt_dam.total_damage() > 0 means took damage
```

Comment:
is it intended?

Reply:
Yea this looks like it got garbled at some point in the past and I fixed it while I was at it.